src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderResourceIntTest_getOffenderAlerts.java

```diff
@@ -69,6 +85,7 @@ void willReturnAllActiveAlertsWhenNoFilter() {
 
             assertThat(jsonContent).extractingJsonPathStringValue("[1].alertType").isEqualTo("X");
             assertThat(jsonContent).extractingJsonPathStringValue("[1].alertCode").isEqualTo("XTACT");
+            assertThat(jsonContent).extractingJsonPathStringValue("[1].alertCodeDescription").isEqualTo("XTACT");
```

Comment:
Could we do wit ha similar assertion for the alertType description (or maybe it's covered elsewhere)?

Reply:
Unit test elsewhere - just didn't have the data for a alertType in an integration test that does not exists